modules/intel-gaudi-ai-accelerator-integration.adoc

```diff
@@ -0,0 +1,41 @@
+:_module-type: CONCEPT
+
+[id='intel-gaudi-ai-accelerator-integration_{context}']
+= Intel Gaudi AI Accelerator integration
+
+[role='_abstract']
+To accelerate your high-performance deep learning (DL) models, you can integrate Intel Gaudi AI accelerators in {productname-short}. {productname-short} also includes the HabanaAI workbench image, which is pre-built and ready for your data scientists to use after you install or upgrade {productname-short}. 
+
+Before you can enable Intel Gaudi AI accelerators in {productname-short}, you must install the necessary dependencies and the version of the HabanaAI Operator that matches the Habana version of the HabanaAI workbench image in your deployment. This allows your data scientists to use Habana libraries and software associated with Intel Gaudi AI accelerators from their workbench. 
+
+For more information about how to enable your OpenShift environment for Intel Gaudi AI accelerators, see link:https://docs.habana.ai/en/v1.10.0/Orchestration/HabanaAI_Operator/index.html[HabanaAI Operator v1.10 for OpenShift] and link:https://docs.habana.ai/en/v1.13.0/Orchestration/HabanaAI_Operator/index.html[HabanaAI Operator v1.13 for OpenShift].
+
+[IMPORTANT]
+====
+Currently, Intel Gaudi AI Accelerator integration is only supported in OpenShift {ocp-minimum-version}. 
+
+You can use Intel Gaudi AI accelerators on {productname-short} with versions 1.10.0 and 1.13.0 of the Habana AI Operator. The version of the HabanaAI Operator that you install must match the Habana version of the HabanaAI workbench image in your deployment. This means that only one version of HabanaAI workbench image will work for you at a time.
```

Comment:
Regarding this sentence: 

The version of the HabanaAI Operator that you install must match the **Habana version** of the HabanaAI workbench image in your deployment.

Do we really need to state "Habana version", and not just "version"?

Reply:
I'd say it's clearer to be specific, so there's no confusion around the actual workbench version (e.g. 2024a).